addons/bus/static/src/multi_tab_service.js

```diff
@@ -162,6 +162,10 @@ export const multiTabService = {
             }
         }
 
+        function isOnLastTab() {
+            return Object.keys(getItemFromStorage("lastPresenceByTab", {})).length < 1; // Main tab is not included.
+        }
+
```

Comment:
I'd rather having a getter in the returned API instead. Also here `< 1` means `0` so we better write it explicitly.